lib/sidekiq/manager.rb

```diff
@@ -68,12 +68,19 @@ def stop(deadline)
       sleep PAUSE_TIME
       return if @workers.empty?
 
-      logger.info { "Pausing to allow workers to finish..." }
+      logger.info { "Pausing to allow workers to finish by #{deadline}..." }
```

Comment:
the `deadline` seems like it's CLOCK_MONOTONIC + options[:timeout], so it might be useful to include the current CLOCK_MONOTONIC value here

https://github.com/braze-inc/sidekiq/blob/0f7f5047cf852ea3c9f0bdcb8b46bac9b22215ec/lib/sidekiq/launcher.rb#L50-L56